Flavours/stellio-flavour/Dockerfile

```diff
@@ -0,0 +1,92 @@
+### specify the node base image with your desired version node:<version>
+FROM ubuntu:18.04
+
+### especially this disables the tzdata interactive asking
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+RUN apt-get -y upgrade
+RUN apt-get install -y wget gnupg2 openjdk-11-jdk
+
+### download Stellio
+RUN apt-get install -y git
+#remember in dockerfile 'cd' only lasts for 1 single line of RUN, use WORKDIR
+WORKDIR /app
+RUN git clone https://github.com/stellio-hub/stellio-context-broker.git
+WORKDIR /app/stellio-context-broker
+
+### compile and assemble Stellio
+RUN ./gradlew build --exclude-task test
+
+### move created jars outside
+RUN mkdir ../stellio/
+RUN cp api-gateway/build/libs/api-gateway-0.5.0.jar ../stellio/
```

Comment:
Rather make it agnostic of the version (you don't want to change this Dockerfile each time we change the version of Stellio, don't you?)

Reply:
indeed

Reply:
as you have a fresh build, you can just do `RUN cp .../api-gateway-*.jar ../stellio/`